tests/scenes/atmosphere/test_particle_layer.py

```diff
@@ -95,3 +98,45 @@ def test_particle_layer_radprops(mode_mono, test_dataset):
     assert all([coord in ds.coords for coord in expected_coords]) and all(
         [var in ds.data_vars for var in expected_data_vars]
     )
+
+
+@pytest.mark.parametrize("wavelength", np.linspace(500.0, 1500.0, 11) * ureg.nm)
```

Comment:
No need to parametrise this test vs wavelength IMO, I'd rather have it vectorised and have a parametrisation vs optical thickness at 550 nm. (Actually, `eval_radpros()` isn't vectorised yet, so we're not truly benefiting from a vector implementation, but it's a good habit to have IMO).

Reply:
I agree!